tests/phpunit/tests/functions/_wpCheckExistingFileNames.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @group functions.php
+ * @covers ::wp_is_numeric_array
+ */
+class Tests_Functions__wp_check_existing_file_names extends WP_UnitTestCase {
+
+	/**
+	 * @dataProvider data__wp_check_existing_file_names
+	 *
+	 * @ticket 53971
```

Comment:
```suggestion
	 * @ticket 55192
```
- Corrected ticket number.